src/main/java/com/bbaga/githubscheduledreminderapp/domain/sources/github/filters/PathFilter.java

```diff
@@ -0,0 +1,44 @@
+package com.bbaga.githubscheduledreminderapp.domain.sources.github.filters;
+
+import com.bbaga.githubscheduledreminderapp.domain.configuration.sources.filters.AbstractFilterConfig;
+import com.bbaga.githubscheduledreminderapp.domain.configuration.sources.filters.PathFilterConfig;
+import com.bbaga.githubscheduledreminderapp.infrastructure.github.GitHubIssue;
+import com.bbaga.githubscheduledreminderapp.infrastructure.github.GitHubPullRequest;
+
+import java.io.IOException;
+
+public class PathFilter implements IssueFilterInterface {
+    private PathFilterConfig config;
+
+    @Override
+    public void configure(AbstractFilterConfig config) {
+        this.config = (PathFilterConfig) config;
+    }
+
+    @Override
+    public Boolean filter(GitHubIssue issue) {
+        // Issues don't contain files, so ignore this filter
+        return false;
```

Comment:
Please double-check that all of my return values in this class make sense :)

Reply:
I think I figured it out - it's the opposite of PHP's filters which I'm most used to 🙃